src/ray/common/ray_config_def.h

```diff
@@ -98,18 +98,19 @@ RAY_CONFIG(bool, lineage_pinning_enabled, false)
 /// See also: https://github.com/ray-project/ray/issues/14182
 RAY_CONFIG(bool, preallocate_plasma_memory, false)
 
+/// We place a soft cap on the number of tasks of a given scheduling class that
+/// can run at once. After the specified interval, a the new task above that
+/// cap is allowed to run. The time before the next tasks (above the cap) are
+/// allowed to run increases exponentially. The soft cap is needed to prevent
+/// deadlock in the case where a task begins to execute and tries to `ray.get`
+/// another task of the same class.
+RAY_CONFIG(int64_t, worker_cap_initial_backoff_delay_ms, 100)
```

Comment:
```suggestion
RAY_CONFIG(int64_t, worker_cap_initial_backoff_delay_ms, 1000)
```

I think 100ms is still too fast; we should have it be at least as long as it takes to start a new Python process.